test/evergreen/find_cmake.sh

```diff
@@ -14,6 +14,9 @@ find_cmake ()
 {
     if [ -n "$CMAKE" ]; then
         return 0
+    elif [ -f "/opt/mongodbtoolchain/v4/bin/cmake" ]; then
```

Comment:
@korteland, if we have the following line for each of our build variants, is it required?
```
PATH=/opt/mongodbtoolchain/v4/bin:$PATH
```

Reply:
In theory, no. But in practice, many of our buildvariants don't include this, even though we want to use this newer cmake (instead of whatever happens to be in `/opt/cmake/` or wherever) basically all of the time.

(This does indicate an alternative approach where we set the path for each task, but that's a much larger job.)

Reply:
Sounds good, thanks for the reply!